package.json

```diff
@@ -38,7 +38,7 @@
     "eslint-config-crowdstrike": "link:",
     "eslint-config-crowdstrike-node": "^3.0.0",
     "eslint-plugin-json-files": "^4.0.0",
-    "eslint-plugin-node": "^11.0.0",
+    "eslint-plugin-n": "^14.0.0",
```

Comment:
Why is this in here again? When i run `yarn why eslint-plugin-n` there is no match and its not consuming any of its rulesets directly?